types/ajv-async/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for ajv-async 1.0
+// Project: https://github.com/epoberezkin/ajv-async#readme
+// Definitions by: es <https://github.com/dmitriismitnov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Ajv } from 'ajv';
+
+declare function ajvAsync(ajv: Ajv): void;
```

Comment:
Looks like this should return `Ajv`? https://github.com/epoberezkin/ajv-async/blob/934e0ccc8e08fc9e78c20560bccf065c8591f4f4/index.js#L17

Reply:
Yes you are right. I fixed it